### PR TITLE
BUG: Open file fix cdaweb

### PR DESCRIPTION
**Pull request by rstoneback**
# Description

Addresses # (issue)

Found warnings about open files in the TravisCI testing log when reviewing pull #722 

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

TravisCI

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

**Comment by rstoneback** [comment]
I added a pull over in pysatNASA: https://github.com/pysat/pysatNASA/pull/47

**Comment by rstoneback** [comment]
Tests are timing out.

File: .travis.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ before_script:
 
 # command to run tests
 script:
- - nosetests -vs --with-coverage --cover-package=pysat
+ - nosetests -q --with-coverage --cover-package=pysat
```
Comment (rstoneback) [inline .travis.yml:63]:
```suggestion
 - nosetests -vs --with-coverage --cover-package=pysat
```

**Comment by aburrell** [comment]
So, what was the warning, and how did using `with` change it?  That was the only real change, right?

**Comment by rstoneback** [comment]
> So, what was the warning, and how did using `with` change it? That was the only real change, right?

Review of test logs in pysat v2.3 revealed:
```
ResourceWarning: Enable tracemalloc to get the object allocation traceback
1733/home/travis/build/pysat/pysat/pysat/instruments/methods/nasa_cdaweb.py:339: ResourceWarning: unclosed file <_io.BufferedWriter name='/tmp/tmpe4lbgacq/timed/saber/timed_l2av207_saber_201901010209_v01.cdf'>
1734  open(saved_local_fname, 'wb').write(req.content)
1735
```

The `with` ensures that any exit code for the attached object is run at the end of the local block, no matter what. For a file object it gets closed.

Adding the `with` is the only real change. I thought the system would recognize your commits as already being in.

**Review by aburrell — APPROVED** [review]

**Pull request merged**